src/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="APPLE A1419 iMac 27"
+      >
```

Comment:
```suggestion
      <img
        src="images/imac.jpeg"
        class="card__image"
        alt="APPLE A1419 iMac 27"
      >
```
indentation 2 spaces for one level nesting